### PR TITLE
Set toggle when migrating domain

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

File: corehq/apps/export/utils.py [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 from dimagi.utils.couch.undo import DELETED_SUFFIX
 from dimagi.utils.modules import to_function
+from toggle.models import Toggle
```
Comment (dimagimon) [inline corehq/apps/export/utils.py]: F401 'Toggle' imported but unused

File: corehq/apps/export/utils.py [hunk 2]
```diff
@@ -2,8 +2,12 @@
...
+from toggle.shortcuts import clear_toggle_cache, set_toggle
```
Comment (dimagimon) [inline corehq/apps/export/utils.py]: F401 'clear_toggle_cache' imported but unused

**Comment by orangejenny** [comment]
Tests don't seem to have run, any idea why? Looks fine, though.

Might be nice to add a note to the toggle at https://github.com/dimagi/commcare-hq/blob/master/corehq/toggles.py#L765 to specify that this toggle should only be turned on if the migration is done (assuming that's true).

**Comment by benrudolph** [comment]
That actually isn't true. you can turn on the toggle anytime you want and it will lazily do the migration as you edit your exports

**Comment by orangejenny** [comment]
Nice :)

**Comment by benrudolph** [comment]
closing in favor of https://github.com/dimagi/commcare-hq/pull/12687

**Pull request closed without merging**